src/main/resources/assets/js/app/view/context/widget/version/VersionsView.ts

```diff
@@ -199,9 +199,13 @@ export class VersionsView
                 .onExecuted((action: api.ui.Action) => {
                     if (!isActive) {
                         new RevertVersionRequest(item.id, this.getContentId().toString()).sendAndParse().then(
-                            (contentId: ContentId) => {
-                                api.notify.NotifyManager.get().showFeedback(i18n('notify.version.changed', item.id));
-                                new ActiveContentVersionSetEvent(this.getContentId(), item.id).fire();
+                            (contentVersionId: string) => {
+                                if (contentVersionId === this.activeVersion.id) {
+                                    api.notify.NotifyManager.get().showFeedback(i18n('notify.revert.no.changes'));
```

Comment:
* Change the key _notify.revert.no.changes_ to _notify.revert.noChanges_
* Remove this phrase from all phrase resource files, except for `phrases.properties` (but change the key there) and you can also translate it to russian in _phrases_ru.properties_. You should never add english text into non-english resource files, because then these new phrases will not be fetched when we will be translating texts before the next release.